app/controllers/eventdetail.controller.js

```diff
@@ -22,6 +22,13 @@
     vm.getEvent = function(id) {
       eventService.getEvent(id)
         .success(function(events) {
+          var current = window.location.href;
+          current = current.split("evnt=")[1];
+          events.data[0].date = events.data[0].date.filter(function(d){
+            if(current == d.start_unix){
```

Comment:
Lines 28-30 can be changed to one single line:
```
return currentStartDate == d.start_unix;
```
Which basically return true/false depending on the evaluation of the comparison.

Reply:
Oh look at you with your fancy code.